recipes/packages.rb

```diff
@@ -118,35 +118,8 @@
                   'AIRFLOW_HOME' => node['airflow']['base_dir']})
     code <<-EOF
       set -e
-      #{node['conda']['base_dir']}/envs/airflow/bin/pip install --no-cache-dir apache-airflow["#{operator}"]==#{node['airflow']['version']}
+      #{node['conda']['base_dir']}/envs/airflow/bin/pip install apache-airflow["#{operator}"]==#{node['airflow']['version']} --constraint https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.6.txt
```

Comment:
This is going to break air-gapped installations - I suggest you download this file and commit it in the repo itself.

Reply:
fxd